metadata-ingestion/src/datahub/ingestion/glossary/classification_mixin.py

```diff
@@ -281,7 +281,7 @@ def get_columns_to_classify(
                     ),
                     values=(
                         sample_data[schema_field.fieldPath]
-                        if schema_field.fieldPath in sample_data.keys()
+                        if schema_field.fieldPath in sample_data
```

Comment:
is this related to SIM118? isn't that one ignored?

```
    "SIM118", # Use key {operator} dict instead of key {operator} dict.keys()
```

Reply:
In some cases it does do a auto-fix